CONTRIBUTING.md

```diff
@@ -239,30 +239,9 @@ the following steps.
 1. Follow the instructions above for creating a working copy of this repository.
 
 2. Follow the [contributing instructions](https://github.com/flutter/engine/blob/master/CONTRIBUTING.md)
-   for the engine repository to create a working copy of the engine. When you
-   create the `.gclient` file for the engine, be sure to create it in a
-   directory named `engine` that is a sibling of the directory in which you
-   cloned this repository. For example, if you cloned this repository into the
-   `/foo/bar/flutter` directory, you should create the `.gclient` file in the
-   `/foo/bar/engine` directory. The actual code from the engine repository will
-   end up in `/foo/bar/engine/src` because `gclient` creates a `src` directory
-   underneath the directory that contains the `.gclient` file.
-
-3. To run tests on your host machine, build one of the host configurations
-   (e.g., `out/host_debug_unopt`). To run examples on Android, build one of the
-   Android configurations (e.g., `out/android_debug_unopt`).
-   When running on the device with `--preview-dart-2` flag you will still need
-   to build corresponding host configuration (e.g., `out/host_debug_unopt` if you
-   are using `out/android_debug_unopt`, `out/host_release` if you use `out/android_release`).
-   Host configuration provides standalone dart sdk for the engine, that is used
-   to run engine dart scripts on the host.
-
-You should now be able to run the tests against your locally built
-engine using the `flutter test --local-engine=host_debug_unopt` command. To run
-one of the examples on your device using your locally built engine, use the
-`--local-engine=android_debug_unopt` option to the `flutter` tool:
-
- * `flutter run --local-engine=android_debug_unopt`
+   for the engine repository to create a working copy of the engine. The instructions
+   also explain how to use the locally built engine instead of the one bundled with
```

Comment:
nit: s/the locally built/a locally-built/

Reply:
Done.